src/main/java/racingcar/domain/RacingCars.java

```diff
@@ -0,0 +1,59 @@
+package racingcar.domain;
+
+import static racingcar.constant.Constant.BOUND;
+import static racingcar.constant.Constant.COLON;
+import static racingcar.constant.Constant.ENTER;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringJoiner;
+
+public class RacingCars {
+
+    private final List<RacingCar> racingCars;
+
+    public RacingCars(List<RacingCar> racingCars) {
+        this.racingCars = racingCars;
+    }
+
+    public void raceOnce(RandomService randomService) {
+        for (RacingCar racingCar : this.racingCars) {
+            racingCar.move(randomService.nextInt(BOUND));
+        }
+    }
+
+    public String createGameResult() {
+        StringJoiner stringJoiner = new StringJoiner(ENTER);
+        for (RacingCar racingCar : this.racingCars) {
+            stringJoiner.add(racingCar.getCarName() + COLON + racingCar.movingDistance());
+        }
+        return stringJoiner.add(ENTER).toString();
+    }
+
+    public List<String> findWinners() {
+        List<String> winners = new ArrayList<>();
+        long finishLine = findMaxPosition();
+        for (RacingCar racingCar : this.racingCars) {
+            addWinnerTo(racingCar, finishLine, winners);
+        }
+        return winners;
+    }
```

Comment:
Stream에 대해 알아보시면 이것저것 개선해볼만한 부분이 많이 나올것 같습니다.

```suggestion
    public List<String> findWinners() {
        long finishLine = findMaxPosition();
        return this.racingCars.stream()
                .filter(it -> it.isLocated(finishLine))
                .map(RacingCar::getCarName)
                .collect(Collectors.toList());
    }
```